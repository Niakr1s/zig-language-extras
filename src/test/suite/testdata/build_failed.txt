src/main.zig:33:19: error: error is ignored
src/main.zig:33:19: note: consider using 'try', 'catch', or 'if'