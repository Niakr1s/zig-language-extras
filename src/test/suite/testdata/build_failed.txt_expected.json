[
  {
    "file": "/project/root/src/main.zig",
    "line": 32,
    "column": 18,
    "message": "error is ignored",
    "severity": 0
  },
  {
    "file": "/project/root/src/main.zig",
    "line": 32,
    "column": 18,
    "message": "consider using 'try', 'catch', or 'if'",
    "severity": 3
  }
]